controller/collaborators.go

```diff
@@ -136,6 +170,10 @@ func (c *CollaboratorsController) AddMany(ctx *app.AddManyCollaboratorsContext)
 
 // Remove user from the list of space collaborators.
 func (c *CollaboratorsController) Remove(ctx *app.RemoveCollaboratorsContext) error {
+	if !c.config.IsAuthorizationEnabled() {
+		// Ignore if authZ is disabled (by default in Dev Mode)
```

Comment:
Maybe log a warning or something in these cases. Just so people don't wonder too much why it does nothing...

Reply:
Good point. Added a warning - https://github.com/fabric8-services/fabric8-wit/pull/1589/commits/ea969c2bbe6d4eca7c30a1f87c80f0b334ffba9a